presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -645,6 +645,19 @@ public void testCast()
         assertQuery("SELECT try_cast(x AS BIGINT) FROM (VALUES 1, 2, 3, NULL) t (x)", "VALUES 1, 2, 3, NULL");
     }
 
+    @Test
+    public void testFormat()
+    {
+        assertQuery("select format('%.6f', sum(1000000 / 1e6))", "SELECT 1.000000");
+        assertQuery("select format('%.6f', avg(1))", "SELECT 1.000000");
+        assertQuery("select format('%d', count(1))", "SELECT 1");
+        assertQuery("select format('%d', arbitrary(1))", "SELECT 1");
+        assertQuery("select format('%s %s %s %s %s', sum(1), avg(1), count(1), max(1), min(1))", "SELECT '1 1.0 1 1 1'");
+        assertQuery("SELECT format('%s', approx_distinct(1.0))", "SELECT 1");
+
+        assertQuery("select format('%d', cast(sum(totalprice / 1e6) as BIGINT)) from orders", "SELECT CAST(sum(totalprice / 1e6) as BIGINT) from orders");
```

Comment:
Move these to a separate test like this: https://github.com/prestosql/presto/pull/3856/files#diff-d2d0aec46d32c16651d0d701fc7d8891

Reply:
sure, will make the changes in an hour